rules/CodingStyle/Application/UseImportsAdder.php

```diff
@@ -74,42 +77,48 @@ public function addImportsToStmts(
                 break;
             }
 
-            $nodesToAdd = array_merge([new Nop()], $newUses);
+            $position = $key + 1;
 
-            $this->mirrorUseComments($stmts, $newUses, $key + 1);
+            $useComments = $this->getAndRemoveExistingUseComments($stmts, $position);
```

Comment:
Instead of mirroring the existing comments into the new use statements, given that after this process the first statement might be an existing use statement, here we just remove the existing comments and save them